spring-cloud-dataflow-docs/src/main/asciidoc/appendix-howto.adoc

```diff
@@ -75,162 +71,154 @@ SPRING_APPLICATION_JSON='{
 }'
 ----
 
-NOTE: Depending on Spring Cloud Data Flow server implementation, you may have to pass the
-environment properties using the platform specific environment-setting capabilities. For instance,
-in Cloud Foundry, you'd be passing them as `cf set-env SPRING_APPLICATION_JSON`.
+NOTE: Depending on the Spring Cloud Data Flow server implementation, you may have to pass the environment properties by using the platform specific environment-setting capabilities. For instance, in Cloud Foundry, you would pass them as `cf set-env SPRING_APPLICATION_JSON`.
+
+
 
 === Logging
 
-Spring Cloud Data Flow is built upon several Spring projects, but ultimately the dataflow-server is a
-Spring Boot app, so the logging techniques that apply to any link:http://docs.spring.io/spring-boot/docs/current/reference/html/howto-logging.html#howto-logging[Spring Boot]
-application are applicable here as well.
+Spring Cloud Data Flow is built upon several Spring projects, but ultimately the dataflow-server is a Spring Boot app, so the logging techniques that apply to any link:http://docs.spring.io/spring-boot/docs/current/reference/html/howto-logging.html#howto-logging[Spring Boot] application are applicable here as well.
 
+While troubleshooting, the two primary areas where enabling the DEBUG logs could be useful are
 
-While troubleshooting, following are the two primary areas where enabling the DEBUG logs could be
-useful.
+* <<troubleshooting-deployment-logs>>
```

Comment:
So, this would make it  a sub-section to Logging? (I couldn't switch to your branch and build the PR-changeslocally)

Reply:
It'd be better if it a sub-section because in the current docs, they are its own section, and it doesn't carry the continuity.